app/src/main/java/org/mozilla/reference/browser/Components.kt

```diff
@@ -30,8 +32,17 @@ class Components(private val context: Context) {
         )
     }
 
+    // TODO: So I really don't know which "Component" this feature belonged in, so I said screw it and put it there
+    // which is clearly wrong. Anyway, maybe it should be even constructed somewhere else? Needs investigation but ok for now.
+    private val remoteTabsStorage = RemoteTabsStorage()
+    val remoteTabs by lazy {
+        val feat = RemoteTabsFeature(backgroundServices.accountManager, core.store, remoteTabsStorage)
+        feat.start()
```

Comment:
We probably want to wrap the start in an AccountObserver so the feature starts when FxA is initialized. You want to do something identical to the [`FxaPushSupportFeature`](https://github.com/mozilla-mobile/android-components/blob/master/components/feature/accounts-push/src/main/java/mozilla/components/feature/accounts/push/FxaPushSupportFeature.kt#L94).

We have "integration" classes that we use for sticking multiple features or feature + app together. That might be a pattern worth following here.

A list of examples you could reference:
<img width="483" alt="Screen Shot 2020-01-07 at 3 16 18 PM" src="https://user-images.githubusercontent.com/1370580/71926219-d5b12800-3160-11ea-81af-ed2ab13fb69b.png">